scripts/github_issue_template.py

```diff
@@ -0,0 +1,39 @@
+'''
+    git hub issue template
```

Comment:
You probably want to change this from .py to .tmpl so we do not try and run this as a script.

Reply:
ok